src/fides/lib/models/fides_user_permissions.py

```diff
@@ -21,10 +20,16 @@ class FidesUserPermissions(Base):
 
     @property
     def total_scopes(self) -> List[str]:
-        """Returns the total model-level scopes the user has, either 1) scopes they are assigned directly,
-        or 2) Roles they have via their scopes."""
-        all_scopes = self.scopes or []
+        """Returns the scopes the user has inherited via their roles."""
+        all_scopes = []
         for role in self.roles:
             all_scopes += ROLES_TO_SCOPES_MAPPING.get(role, [])
 
         return sorted(list(set(all_scopes)))
+
+    @property
+    def scopes(self) -> List[str]:
+        """Backwards-compatible property for use while the UI is still looking at userpermission.scopes
+        Users can no longer be assigned scopes directly.
+        """
+        return self.total_scopes
```

Comment:
For backwards compatibility if the UI is still using FidesUserPermission.scopes anywhere. I am returning all scopes they have via their roles (total scopes) but alternatively we could return an empty list.

Reply:
I think [here](https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/features/user-management/PermissionsForm.tsx/#L55-L57) is the only place still referring to `scopes` in the frontend, and that was also a backwards compatibility thing